src/gridstack.ts

```diff
@@ -2493,7 +2447,7 @@ export class GridStack {
     node.el = this.placeholder;
     node._lastUiPosition = ui.position;
     node._prevYPix = ui.position.top;
-    node._moving = (event.type === 'dragstart'); // 'dropover' are not initially moving so they can go exactly where they enter (will push stuff out of the way)
+    node._moving = (event.type === 'dragstart' || event.type === 'resizestart'); // 'dropover' are not initially moving so they can go exactly where they enter (will push stuff out of the way)

```

Comment:
that logic could break other code... might need to check _sizing or something in your code instead. please revert.

Reply:
I've noticed that the "_moving" is already used for both move and resize gestures. However, let's keep them separated for this specific item about overriding the inline style on the fly.